### PR TITLE
Adds CTRL-A CTRL-E support to LineEdit on macOS

### PR DESCRIPTION
**Pull request by timothyqiu**
These two shortcuts are commonly available on macOS.

### CODE REVIEW

File: doc/classes/LineEdit.xml [hunk 1]
```diff
@@ -22,6 +22,8 @@
 		- Ctrl + N: Like the down arrow key, move the cursor to the next line
 		- Ctrl + D: Like the Delete key, delete the character on the right side of cursor
 		- Ctrl + H: Like the Backspace key, delete the character on the left side of the cursor
+		- Ctrl + A: Like the Home key, move the cursor to the beginning of the line
```
Comment (Calinou) [inline doc/classes/LineEdit.xml]: This will conflict with the "select all" key binding present on other platforms.

**Comment by Calinou** [comment]
Good change :slightly_smiling_face:

Should this also be added to TextEdit, or is it already present there?

**Comment by timothyqiu** [comment]
It's already there, but in a more "text editor" fashion. CTRL-A toggles the cursor between the first character and the first non-whitespace character.

**Comment by akien-mga** [comment]
Thanks!

**Pull request merged**